Gemfile.lock

```diff
@@ -28,38 +28,42 @@ GEM
     coffee-script-source (1.11.1)
     colorator (1.1.0)
     colorize (0.8.1)
-    commonmarker (0.17.7)
+    commonmarker (0.17.9)
       ruby-enum (~> 0.5)
     concurrent-ruby (1.0.5)
+    em-websocket (0.5.1)
+      eventmachine (>= 0.12.9)
+      http_parser.rb (~> 0.6.0)
     ethon (0.11.0)
       ffi (>= 1.3.0)
+    eventmachine (1.2.5)
     execjs (2.7.0)
-    faraday (0.13.1)
+    faraday (0.14.0)
       multipart-post (>= 1.2, < 3)
-    ffi (1.9.18)
+    ffi (1.9.23)
     forwardable-extended (2.6.0)
     gemoji (3.0.0)
-    github-pages (172)
+    github-pages (180)
       activesupport (= 4.2.9)
-      github-pages-health-check (= 1.3.5)
-      jekyll (= 3.6.2)
+      github-pages-health-check (= 1.4.0)
+      jekyll (= 3.7.3)
```

Comment:
@geeknoid do we require those changes ? if can you update https://github.com/istio/istio.github.io#working-with-the-site

Reply:
for me bundle install now hangs at
Installing commonmarker 0.17.9 with native extensions

Reply:
it was hanging because I had a go binary called "echo" in my path (!)